docs/AwsMarketplaceGuide.md

```diff
@@ -55,6 +57,27 @@ By subscribing to Scalar products in the AWS Marketplace, you can pull the conta
 
    You need to specify the private container registry (ECR) of the AWS Marketplace and the version (tag) as the value for `[].image.repository` and `[].image.version (tag)` in the custom values file. You also need to specify the service account name that you created in the previous step as the value for `[].serviceAccount.serviceAccountName` and set `[].serviceAccount.automountServiceAccountToken` to `true`.
 
+   * ScalarDB Cluster Examples
```

Comment:
@josh-wong 
I think it's something we should consider applying tabs.
We can do it in another PR. Anyway, let's discuss it.

Reply:
I agree. In other documents, there also are several similar parts that describe configuration examples in each product. I think it is better to use `tabs` in such examples part.

Reply:
Yes, we can definitely add tabs to make parts like this easier to follow. I will consider how to implement tabs in this case (and in similar docs, like for Helm Charts).